WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -903,6 +994,40 @@ + (BOOL)isNewEditorEnabled
 
 #pragma mark - Instance Methods
 
+- (void)keyboardWillHide:(NSNotification*)aNotification
+{
+    // If the format bar tooltip is still hanging around, let's git rid of it
+    if (self.formatBarToolTip) {
+        [self.formatBarToolTip cancelCurrentTooltip];
+        self.formatBarToolTip = nil;
+    }
+}
+
+- (void)keyboardWasShown:(NSNotification*)aNotification
```

Comment:
in the iPhone 5s 8.4 simulator ` +- (void)keyboardWasShown:(NSNotification*)aNotification ` is being called twice for every orientation change.  I haven't been able to suss out why that's happening.